packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -476,6 +477,12 @@ class SizeTransition extends AnimatedWidget {
   /// A value of 0.0 (the default) indicates the center for either [axis] value.
   final double axisAlignment;
 
+  /// The factor to multiply the cross axis size of the child by.
+  ///
+  /// If the value of [crossAxisSizeFactor] is less than one, the child will be
+  /// clipped in the appropriate axis.
+  final double? crossAxisSizeFactor;
```

Comment:
Maybe add a comment describing the default behavior when this is not defined. (The cross axis size is as large as the parent).

Reply:
I added a comment in [chore: Add assert and comment](https://github.com/flutter/flutter/pull/134659/commits/c4b5ca0055e9eb426256025a3bb31ffcb15994b2). Is that good enough?